analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -63,7 +63,7 @@ def create_actions_map(actions, manager):
     return result
 
 
-def __get_analyzer_version(context, analyzer_config_map):
+def __get_analyzer_version(context, analyzer_cfg):
```

Comment:
Could we reduce the number of places we get the version of an analyzer? There is already a [version.py](https://github.com/Ericsson/codechecker/blob/4aac198f547dadc0030f21fbd874c7a76bd9b13b/analyzer/codechecker_analyzer/analyzers/clangsa/version.py) for clangsa.
Maybe we could introduce the same for clang-tidy. PR #2357 has the similar issue.